packages/psss/src/views/Tabs/ProfileTabView.js

```diff
@@ -34,38 +36,82 @@ const SuccessMessage = styled(SmallAlert)`
   margin-bottom: 1.5rem;
 `;
 
+const Divider = styled(MuiDivider)`
+  margin: 1.8rem 0;
+`;
+
+const STATUS = {
+  IDLE: 'idle',
+  LOADING: 'loading',
+  SUCCESS: 'success',
+  ERROR: 'error',
+  DISABLED: 'disabled',
+};
+
 const ProfileTabViewComponent = React.memo(({ user, onUpdateProfile }) => {
+  const [status, setStatus] = useState(STATUS.IDLE);
   const [errorMessage, setErrorMessage] = useState(null);
   const [successMessage, setSuccessMessage] = useState(null);
-  const [isLoading, setIsLoading] = useState(false);
+  const [profileImage, setProfileImage] = useState({
+    fileId: null,
+    data: user.profileImage,
+  });
   const { handleSubmit, register, errors } = useForm();
 
   const onSubmit = handleSubmit(async ({ firstName, lastName, role, employer }) => {
-    setIsLoading(true);
+    setStatus(STATUS.LOADING);
     setErrorMessage(null);
     try {
       await onUpdateProfile({
         first_name: firstName,
         last_name: lastName,
         position: role,
         employer,
+        profileImage: profileImage.fileId && profileImage,
```

Comment:
Pointing me to this line made me notice that it's camel case, with the first two fields in snake case